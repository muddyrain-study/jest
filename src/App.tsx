import TestUI from './components/TestUi';
import Items from './components/items';

function App() {
  return (
    <div className='App'>
      <Items />
      <TestUI />
    </div>
  );
}

export default App;
