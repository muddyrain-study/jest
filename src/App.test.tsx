import { render, screen } from '@testing-library/react';
import App from './App';

test('test-react', () => {
  render(<App />);
  const list = screen.getAllByTestId('list');
  expect(list).toMatchSnapshot();
});
