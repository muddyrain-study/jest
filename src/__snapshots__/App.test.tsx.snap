// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test-react 1`] = `
Array [
  <ul
    data-testid="list"
  >
    <li>
      张三
    </li>
    <li>
      李四
    </li>
    <li>
      王武
    </li>
  </ul>,
]
`;
